Factory/Classes/Facilities/AssemblyLine.m

```diff
@@ -8,6 +8,7 @@
 
 @implementation AssemblyLine
 
+// I thought that the right style: latitude = _latitude
```

Comment:
It's not really "the right style"; it's just Apple's style. Meanwhile Noveo has its own style =)